src/app/member/components/MemberList.tsx

```diff
@@ -0,0 +1,193 @@
+"use client";
+import React from "react";
+import { DataTabel } from "@/components/common/DataTabel/Tabel";
+import { Member } from "@/types";
+import { CaretSortIcon } from "@radix-ui/react-icons";
+import { ColumnDef } from "@tanstack/react-table";
+import { Badge } from "@/components/ui/badge";
+import MemberTableActionRender from "./MemberActionRender";
+
+const membersData: Member[] = [
```

Comment:
Why fake data? Don't we already have DB connection established? Are you blocked by @Mubashar1009 on user creation?